modules/geth/charts.go

```diff
@@ -0,0 +1,182 @@
+package geth
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	Charts = module.Charts
+	Chart  = module.Chart
+	Dims   = module.Dims
+	Dim    = module.Dim
+)
+
+var charts = Charts{
+	chartAncientChainData.Copy(),
+	chartChaidataDisk.Copy(),
+	chartTxPoolPending.Copy(),
+	chartNewBlocksCreation.Copy(),
+	chartTxPoolQueued.Copy(),
+	chartP2PNetwork.Copy(),
+	chartP2PNetworkDetails.Copy(),
+	chartNumberOfPeers.Copy(),
+	chartRpcInformation.Copy(),
+	chartAncientChainDataRate.Copy(),
+	chartChaidataDiskRate.Copy()
+}
+
+var (
+
+	chartAncientChainDataRate = Chart{
+		ID:    "chaindata_ancient_rate",
+		Title: "Ancient Chaindata rate",
+		Units: "bytes",
```

Comment:
```suggestion
		Units: "bytes/s",
```

> Algo: "incremental"

Always means per second rate. Units should have `/s` suffix.